vaadin-archetype-widget/src/main/resources/archetype-resources/__artifactId__-addon/pom.xml

```diff
@@ -112,7 +112,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-javadoc-plugin</artifactId>
-				<version>2.10.3</version>
+				<version>3.0.0</version>
```

Comment:
This version of JavaDoc plugin does not actually work with JDK 10. It uses an old version of commons-lang3.

Issue in JavaDoc plugin tracker https://issues.apache.org/jira/browse/MJAVADOC-518